common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java

```diff
@@ -71,18 +71,18 @@
  */
 @SuppressWarnings("PMD.AbstractClassShouldStartWithAbstractNamingRule")
 public abstract class GrpcClient extends RpcClient {
-    
+
```

Comment:
Please use nacos code style to reformat code.

Reply:
> Please use nacos code style to reformat code.

I try to format the code with nacos check style, but there is no problem, I think it should be that nacos check style does not restrict this

Reply:
![image](https://github.com/alibaba/nacos/assets/48236177/20fa890a-f5af-4342-be9d-e2e3b4b4d0c7)
![image](https://github.com/alibaba/nacos/assets/48236177/73a4e606-44b2-4370-a522-089c7527cf13)

Reply:
Maybe #11021 is a better way.

Reply:
> Please use nacos code style to reformat code.

Done. PTAL